backend/tournesol/views/stats.py

```diff
@@ -0,0 +1,76 @@
+"""
+API endpoints to show public statistics
+"""
+
+from datetime import timedelta
+
+from django.utils import timezone
+from drf_spectacular.utils import extend_schema, extend_schema_view
+from rest_framework import generics
+from rest_framework.permissions import AllowAny
+from rest_framework.response import Response
+
+from core.models import User
+
+from ..models import Comparison, Video
+from ..serializers import StatisticsSerializer
+
+
+class Statistics:
```

Comment:
I'm wondering, should we move this into the models folder or somewhere else?

It's not an entity persisted in the database as is, so it's not a Django model, but it's still a representation of an object that needs to be serialized.

Leaving the class here is very OK for me, because it's small and it's the first of its kind.

Reply:
Let's keep it here for now then.